src/price_data.py

```diff
@@ -563,3 +568,45 @@ def get_cost(
         if isinstance(tr, transaction.SoldCoin):
             return price * tr.sold
         raise NotImplementedError
+
+    def check_database(
+        self,
+    ):
+        for db_path in Path(config.DATA_PATH).glob("*.db"):
+            if db_path.is_file():
+                platform = db_path.stem
+
+                try:
+                    get_price = getattr(self, f"_get_price_{platform}")
+                except AttributeError:
+                    if platform == "coinbase":
+                        get_price = self._get_price_coinbase_pro
+                    else:
+                        raise NotImplementedError(
+                            "Unable to read data from %s", platform
+                        )
+
+                with sqlite3.connect(db_path) as conn:
+                    query = "SELECT name FROM sqlite_master where type='table'"
+                    cur = conn.execute(query)
+
+                    for table in cur.fetchall():
+                        tablename = table[0]
+                        pair = tablename.split("/")
```

Comment:
We could add a assert here, `assert len(pair) == 2, f"Pair must contain exactly two coins, but {len(pair)} given: {pair}`. Or use `base_asset, quote_asset = tablename.split("/")` instead.